lnurlp/lnurlp.py

```diff
@@ -106,4 +106,16 @@ def init(options, configuration, plugin):
     "Path of the webserver's lnurlp-JSON-file"
 )
 
+plugin.add_option(
+    "lnurlp-ratelimit-rate",
+    "2 per minute",
+    "Change flask's ratelimiter, default is 2 per minute"
+)
+
+plugin.add_option(
+    "lnurlp-ratelimit-switch",
+    "enable",
+    "Flask's ratelimiter usage, enabled by default"
```

Comment:
please add the available options to the description here (`disable` and `enable`)